modules/menus/energy/profiles/index.ts

```diff
@@ -1,25 +1,38 @@
 const powerProfiles = await Service.import('powerprofiles');
 import { PowerProfile, PowerProfileObject, PowerProfiles } from 'lib/types/powerprofiles.js';
-import icons from '../../../icons/index.js';
 import { BoxWidget } from 'lib/types/widget.js';
+import icons from '../../../icons/index.js';
+import { uptime } from 'lib/variables.js';
 
 const EnergyProfiles = (): BoxWidget => {
     const isValidProfile = (profile: string): profile is PowerProfile =>
         profile === 'power-saver' || profile === 'balanced' || profile === 'performance';
 
+    function up(up: number): string {
+        const h = Math.floor(up / 60);
+        const m = Math.floor(up % 60);
+        return ` : ${h < 10 ? '0' + h : h}:${m < 10 ? '0' + m : m}`;
+    }
```

Comment:
```suggestion
    function renderUptime(curUptime: number): string {
        const days = Math.floor(curUptime / (60 * 24));
        const hours = Math.floor((curUptime % (60 * 24)) / 60);
        const minutes = Math.floor(curUptime % 60);
        return ` : ${days}d ${hours}h ${minutes}m`;
    }
```
We can add days and adjust the format to make it prettier. Plus verbose and descriptive variable names are always preferred.